cms/templates/base.html

```diff
@@ -1,11 +1,12 @@
 ## coding=utf-8
 <%namespace name='static' file='static_content.html'/>
 <%!
-from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import ugettext as _
```

Comment:
Reviewers: calling attention to this line which adds html escaping by default to all uses of `_()`.  The code is smart enough to not double escape with the page expression_filter below.